www/api/og.tsx

```diff
@@ -0,0 +1,31 @@
+import { ImageResponse } from '@vercel/og';
+import React from 'react';
+
+export const config = {
+  runtime: 'experimental-edge',
+};
+
+export default function () {
```

Comment:
Here, we need to be able to read query parameters and add them to the image below